### PR TITLE
Use UTF8 instead of Console's encoding.

### PR DESCRIPTION
**Pull request by jakubsemerak**
It could be configurable too...

### CODE REVIEW

**Comment by fean** [comment]
Hey man! 🙂 

Thanks for the change! 👍 
Awesome idea! Do you want to pick the config up as well? 
It would be a nice addition if you would have the time! 🙂 

Otherwise I'll pick it up! 👍

**Comment by jakubsemerak** [comment]
Hello 🙂

I was not sure where to put it, so I made it static for now. Any idea where this configuration should be placed? For example in the `PdfOutput` class?

**Comment by fean** [comment]
Yeah, I think that would be a good solution. 👍 Maybe we could let it default to `Encoding.UTF8`

**Comment by jakubsemerak** [comment]
Check it out, I also unified some whitespace inconsistency. Hope its okay 🙂

File: NuGet/Codaxy.WkHtmlToPdf/content/PdfConvert.cs [hunk 1]
```diff
@@ -27,43 +27,47 @@ public class PdfConvertTimeoutException : PdfConvertException
...
+        public Stream OutputStream { get; set; }
         /// <summary>
         /// Callback that will be called once conversion has completed.
         /// </summary>
-		public Action<PdfDocument, byte[]> OutputCallback { get; set; }
+        public Action<PdfDocument, byte[]> OutputCallback { get; set; }
+        /// <summary>
+        /// Encoding of the document.
+        /// </summary>
+        public Encoding Enconding { get; set; } = Encoding.UTF8;
```
Comment (fean) [inline NuGet/Codaxy.WkHtmlToPdf/content/PdfConvert.cs]:
```suggestion
        public Encoding Encoding { get; set; } = Encoding.UTF8;

```

File: NuGet/Codaxy.WkHtmlToPdf/content/PdfConvert.cs [hunk 2]
```diff
@@ -295,7 +299,7 @@ public static void ConvertHtmlToPdf(PdfDocument document, PdfConvertEnvironment
                     process.BeginErrorReadLine();
 
                     if (document.Html != null && IsEmptyUrl(document.Url))
-                        using (var stream = process.StandardInput)
+                        using (var stream = new StreamWriter(process.StandardInput.BaseStream, woutput.Enconding))
```
Comment (fean) [inline NuGet/Codaxy.WkHtmlToPdf/content/PdfConvert.cs]:
```suggestion
                        using (var stream = new StreamWriter(process.StandardInput.BaseStream, woutput.Encoding))

```

**Comment by fean** [comment]
No worries! 🙂 Just a small typo, then lets merge it! 👍

**Comment by jakubsemerak** [comment]
Thanks for the review, it should be ok now 🙂

**Review by fean — APPROVED** [review]

**Pull request merged**